_maps/map_files/stations/cerestation.dmm

```diff
@@ -69933,16 +69929,6 @@
 	icon_state = "white"
 	},
 /area/station/science/misc_lab)
-"pqq" = (
-/obj/machinery/door/airlock/multi_tile/glass{
-	dir = 4
-	},
-/obj/effect/mapping_helpers/airlock/autoname,
-/obj/machinery/door/firedoor,
-/turf/simulated/floor/plasteel{
-	icon_state = "dark"
-	},
-/area/station/public/quantum/security)
```

Comment:
![image](https://github.com/ParadiseSS13/Paradise/assets/77684085/86997700-2963-47d6-bec1-36b481dd023b)

It looks like you removed a multi tile airlock when dealing with the conflict

Reply:
I don't really know how but while it seems to be missing in the map file it still appears in game and StrongDMM?

![still here](https://github.com/ParadiseSS13/Paradise/assets/134484326/3d191e21-9c0e-4a36-bce7-99fe9a5a0728)

Reply:
I will verify why the bot is showing changes over this region

Reply:
Took a good look at it and couldnt spot any difference, you can resolve this.